lib/schema.js

```diff
@@ -78,6 +78,10 @@ const schema = context => {
 
     template: Joi.string().required(),
 
+    'target-branch': Joi.string()
+      .allow(null)
+      .default(DEFAULT_CONFIG['target-branch']),
```

Comment:
See line 8-12
https://github.com/release-drafter/release-drafter/blob/5e307e1c8f08c923e0ae855e21a6faf64bce12ea/lib/schema.js#L8-L12

```suggestion
      .default(defaultBranch),
```

Reply:
This would change the logic and introduce a breaking change.

When we're creating the draft, it's ok to fall back to the default branch, because the draft's target branch should be specified anyway:
https://github.com/release-drafter/release-drafter/blob/c642710689e15269b262c16e6ffd9eaa5740ba39/index.js#L67-L68

But when we're searching for releases, we should either:

- Filter by branch, if there is a `target-branch` config
- Not filter by branch, if there is no `target-branch` config (or it's null), taking any release from any branch into consideration to calculate the version (previous behaviour)

The branch filter is passed in to `findReleases`:
https://github.com/release-drafter/release-drafter/blob/c642710689e15269b262c16e6ffd9eaa5740ba39/index.js#L32

Reply:
Than add to properties, one for `filtered-branch` and one for `target_commitish`

To me this seems like going beyond the use case for release-drafter but I could be wrong.